packages/clerk-js/src/ui/common/authPropHelpers.ts

```diff
@@ -78,3 +78,13 @@ export const buildAuthQueryString = (data: BuildAuthQueryStringArgs): string | n
   }
   return Object.keys(query).length === 0 ? null : qs.stringify(query);
 };
+
+export const normalizeRoutingOptions = ({
+  routing,
+  path,
+}: {
+  routing?: RoutingStrategy;
+  path?: string;
+}): RoutingOptions => {
+  return { routing: !routing && path !== undefined ? 'path' : routing, path } as RoutingOptions;
```

Comment:
I think the check here should be:

```suggestion
  return { routing: !!path && routing === undefined ? 'path' : routing, path } as RoutingOptions;
```

Reply:
`routing` can't really be null so we can just check `!routing` I guess.

Reply:
It shouldn't but it might be, lets test it explicitly here please because we want to avoid changing behavior for as many cases as possible

Reply:
Done! We should change this in the next major version though.